client/my-sites/controller.js

```diff
@@ -112,46 +111,62 @@ function renderNoVisibleSites( context ) {
 	);
 }
 
-function isPathAllowedForDomainOnlySite( pathname, domainName ) {
+function renderSelectedSiteIsDomainOnly( { reactContext, selectedSite } ) {
+	const EmptyContentComponent = require( 'components/empty-content' );
+	const { store: reduxStore } = reactContext;
+
+	removeSidebar( reactContext );
+
+	renderWithReduxStore(
+		React.createElement( EmptyContentComponent, {
+			title: i18n.translate( 'This feature is not available for domains' ),
+			line: i18n.translate( 'To use this feature you need to create a site' ),
+			action: i18n.translate( 'Create New Site' ),
+			actionURL: '//dashboard.wordpress.com/wp-admin/index.php?page=my-blogs',
```

Comment:
Shouldn't we redirect users to https://wordpress.com/start/ instead? That's what we do for [this button](https://github.com/Automattic/wp-calypso/blob/c6fc8b691db964d3e71b0e857c3714cebac99149/client/my-sites/sidebar/sidebar.jsx#L710) for example.